app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    cp, file1, file2 = command.split()
+    if cp != "cp":
+        return
+    if file1 == file2:
+        return
+
+    with open(file1, "r") as f1, open(file2, "w") as f2:
+        f2.write(f1.read())
```

Comment:
Code Efficiency: You're reading the entire source file into memory and then writing it to the destination file. This could cause problems if the source file is very large. It would be more efficient to read and write the file in chunks.